test/JoeMakerV4.test.ts

```diff
@@ -155,7 +155,7 @@ describe("joeMakerV4", function () {
     it("should convert USDC - DAI", async function () {
       await this.zap.zapIn(this.usdcDai.address, { value: ethers.utils.parseEther("2") })
       await this.usdcDai.transfer(this.joeMakerV4.address, await this.usdcDai.balanceOf(this.dev.address))
-      await this.joeMakerV4.convert(this.dai.address, this.usdc.address)
+      await this.joeMakerV4.convert(this.dai.address, this.usdc.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdcDai.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("214878919")
```

Comment:
Again, I think we need better assertions here; the expectation is that Bar receives a certain amount of JOE